app/helpers/employees_helper.rb

```diff
@@ -0,0 +1,2 @@
+module EmployeesHelper
+end
```

Comment:
If you don't occupy this file, delete it.

Reply:
done.